lib/mixins/property-effects.html

```diff
@@ -86,7 +89,7 @@
    * Runs all effects of a given type for the given set of property changes
    * on an instance.
    *
-   * @param {Object} inst The instance with effects to run
+   * @param {PropertyEffectsInstance | Polymer_PropertyEffects} inst The instance with effects to run
```

Comment:
Looks like I need a bunch of casts now because a lot of methods in `PropertyEffects` class call out to these functions and the type definition doesn't really exist